data/competencies.yml

```
@@ -633,9 +634,11 @@
 # Senior 1 to Senior 2, Technical
 
 - id: 97ac147f-1b61-43c4-b677-dbb837ba092b
-  summary: Makes pragmatic decisions about technical trade-offs
+  summary: Makes pragmatic decisions about technical trade-offs beyond their project
   examples:
-    - Can predict and explain when something is worth refactoring even when it will impact the speed of delivery
+    - Can articulate why the overhead of using a third party system is worth it for their project
+    - Decides to invest time in building a dashboard for stakeholders to reduce the number of queries they make to the team
+    - Contributes to buy vs. build vs. blend discussions impacting their project
```

```suggestion
    - Gathers relevant data to inform buy vs. build vs. blend discussions impacting their project
```

Feel it needs something more concrete than 'contributes'